python/chronos/src/bigdl/chronos/metric/forecast_metrics.py

```diff
@@ -259,3 +267,104 @@ def get_latency(func, *args, num_running=100, **kwargs):
+               which represents random number.
+        :param layout: a 2-dim tuple, indicate the row_num and col_num to plot.
+        :param prediction_internval: a float, indicates the confidence percentile. Default to
+               0.95 refer to 95% confidence. This only effective when `std` is not None.
+        :param figsize: figure size to be inputed to pyplot. Default to (16,16).
+        :param output_file: a path, indicates the save path of the output plot. Default to
+               None, indicates no output file is needed.
+        :param **kwargs: other paramters will be passed to matplotlib.pyplot.
+        '''
+        import matplotlib.pyplot as plt
+
+        # shape check
+        if std is not None:
+            _check_shape(y, std, "y", "std")
+        if ground_truth is not None:
+            _check_shape(y, ground_truth, "y", "ground_truth")
+
+        batch_num = y.shape[0]
+        horizon = y.shape[1]
+        lookback = 0 if x is None else x.shape[1]
+        row_num = 1 if layout is None else layout[0]
+        col_num = 1 if layout is None else layout[1]
+
+        y_index = list(range(lookback, horizon + lookback))
+        x_index = list(range(0, lookback))
+        iter_num = 1
+        instance_index_iter = iter(instance_index) if instance_index is not None else None
+
+        plt.figure(figsize=figsize, **kwargs)
+
+        for row_iter in range(1, row_num + 1):
+            for col_iter in range(1, col_num + 1):
+                # generate the index
+                if instance_index_iter is None:
+                    instance_index = random.randint(0, y.shape[0])
+                else:
+                    try:
+                        instance_index = next(instance_index_iter)
+                    except e:
+                        # nothing to plot, skip following grids
+                        continue
+                ax = plt.subplot(row_num*100 + col_num*10 + iter_num)
+                ax.plot(y_index, y[instance_index, :, feature_index], color="royalblue")
+                if ground_truth is not None:
+                    ax.plot(y_index, ground_truth[instance_index, :, feature_index],
+                            color="limegreen")
+                if x is not None:
+                    ax.plot(x_index, x[instance_index, :, feature_index], color="black")
+                    ax.plot([x_index[-1], y_index[0]],
+                            np.array([x[instance_index, -1, feature_index],
+                                    y[instance_index, 0, feature_index]]),
+                            color="royalblue")
+                    if ground_truth is not None:
+                        ax.plot([x_index[-1], y_index[0]],
+                                np.array([x[instance_index, -1, feature_index],
+                                        ground_truth[instance_index, 0, feature_index]]),
+                                color="limegreen")
```

Comment:
I wonder this plot is used for?

Reply:
to connect x and y, the plot only contains 2 dots, 1 for the last value of x and 1 fore the first value of y